articles/libraries/secure-local-development.md

```diff
@@ -0,0 +1,74 @@
+---
+section: libraries
+description: Securing local development servers to work with samesite cookies
+topics:
+  - libraries
+  - samesite
+contentType:
+  - guide
+---
+
+# Secure Local Development
+Commonly developers run local web servers on non-secure channels (http).  Generally because the perceived difficulty of running a secure local server (https), and/or it is not a requirement for their local setup.  This guide will discuss when you should run a secure local server `https` and provide simple instructions for setting it up.
```

Comment:
First two sentences here need work, seems like it should be a single sentence.

Reply:
Also, you probably want to settle on backticks or no for http/`http`/https/`https`